docker-deploy/docker-compose.yml

```diff
@@ -16,62 +16,82 @@ services:
           memory: 2048M
     volumes:
       - ./devtools:/app/devtools
-      - ./logs1:/app/logs
+      - ./logs:/app/logs
     networks:
       - axon-net
-    command: ./axon -c=/app/devtools/chain/node_1.toml -g=/app/devtools/chain/genesis.json
+    command: ./axon -c=/app/devtools/chain/config.toml -g=/app/devtools/chain/genesis.json
 
-  axon2:
-    container_name: axon2
-    image: axonweb3/axon:amd64-v0.1.0-beta.3
-    restart: always
-    privileged: true
-    deploy:
-      resources:
-        limits:
-          cpus: '1'
-          memory: 2048M
+
+  faucet-server:
+    image: ghcr.io/simon-tl/axon-faucet:latest
```

Comment:
image tag it's  recommended to specify the version, not the latest.

Reply:
ok, this is just me testing using